kie-dmn-feel/src/main/java/org/kie/dmn/feel/parser/feel11/FEELParser.java

```diff
@@ -46,26 +49,35 @@ public static FEEL_1_1Parser parse(FEELEventListenersManager eventsManager, Stri
     }
 
     public static boolean isVariableNameValid( String source ) {
+        return checkVariableName( source ).isEmpty();
+    }
```

Comment:
Not sure if we need to keep the method above or simply remove it and always use the method below that returns a list of errors instead of a boolean?